app/controllers/spree/api/affirm_controller.rb

```diff
@@ -1,7 +1,11 @@
 module Spree
   module Api
     class AffirmController < Spree::Api::BaseController
+      include Spree::Core::ControllerHelpers::Order
+      include Spree::Core::ControllerHelpers::Auth
```

Comment:
If we're using an API controller, we should not be using the session for authorization. API's that use sessions are vulnerable to CSRF. Perhaps this should be a regular controller instead?